### PR TITLE
Enable merchant

### LINKED ISSUE #31: User Story 42, Admin disables a merchant account
```
As an admin merchant
When I visit the merchant index page
And I click on a "disable" button for an enabled merchant
I am returned to the admin's merchant index page
And I see a flash message that the merchant's account is now disabled
And I see that the merchant's account is now disabled
This merchant cannot log in
```

DO THESE STORIES FIRST:
- User Story 41, Admin visits Merchant Index Page

### Checklist:

- [ ] base tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #35

child of #82

depends on #32

### PR DESCRIPTION
**Pull request by mikekarnes123**
- [] Check this if the PR has been approved by the team to be a quick patch and the rest of this form will not be filled out

# Test Coverage Percentage (Model/Feature):
- rspec - 100%
- rspec spec/models - 100%

# What functionality does this accomplish?
Allows merchants and subsequently their items to be enabled 

# What did you struggle on to complete?

# Implements/Fixes:
* description
closes #31 

# Testing Changes
- [] No Tests have been changed
-> Some Tests have been changed
- [] All of the Tests have been changed(Please describe what in the world happened)

# Checklist:

- [] My code has no unused/commented out code
- [] I have reviewed my code
-> I have commented my code, particularly in hard-to-understand areas
- [] I have fully tested my code

# Please include an emoji of how you feel about this branch:

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -26,6 +26,10 @@ def disable_merchant_items
     items.each {|item| item.disable}
   end
 
+  def enable_merchant_items
+    items.each {|item| item.enable}
```
Comment (milevy1) [inline app/models/user.rb]: Can we refactor to `items.update_all(enabled: true)`

This would reduce it to just one ActiveRecord database call instead of one per item
